### DISCUSSION TITLE
Flux documentation for Accodion

### DISCUSSION THREAD

**Discussion by kyledoesdev** [livewire/flux#743] [Feature Requests]
Currently the documentation for Accordion misses the variant of `reverse`. I didn't know it was an option until I published the component and saw in the `heading.blade.php` file that there was a match key for reverse. 

I believe it would be beneficial to add this.

**Comment by eviera** [comment]
I have the same issue with the documentation for `input`, as I have to resort to the `vendor/livewire/flux-pro/stubs/resources/views/flux/input/index.blade.php` to see that there are two `variants`, `filled `and `outline`

Would you plan to publish full documentation for the components ?
